xbmc/cores/VideoPlayer/DVDCodecs/Video/DVDVideoCodec.h

```diff
@@ -68,7 +68,9 @@ struct VideoPicture
   bool hasLightMetadata = false;
   AVContentLightMetadata lightMetadata;
 
-  AVPixelFormat pixelFormat; //< source pixel format
+  bool isHdr{false}; //< true if source is HDR (hdrType != HDR_TYPE_NONE)
```

Comment:
what about a name like isHdrTransfer. That makes the change in CProcessorHD::IsSuperResolutionSuitable more readable.